gssapi/raw/ext_dce.pyx

```diff
@@ -8,15 +8,34 @@ from gssapi.raw.sec_contexts cimport SecurityContext
 
 from gssapi.raw.misc import GSSError
 from gssapi.raw import types as gssapi_types
-from gssapi.raw.named_tuples import IOVUnwrapResult, WrapResult, UnwrapResult
+from gssapi.raw.named_tuples import IOVUnwrapResult
 from collections import namedtuple
 from collections.abc import Sequence
 
 from enum import IntEnum
 from gssapi.raw._enum_extensions import ExtendableEnum
 
+# Kept for backwards compatibility - functions used to be declared here
+try:
+    from gssapi.raw.ext_dce_aead import wrap_aead, unwrap_aead
+except ImportError:
+    pass
+
 
 cdef extern from "python_gssapi_ext.h":
+    """
+    #ifdef OSX_HAS_GSS_FRAMEWORK
+    #define gss_wrap_iov(t, u, v, w, x, y, z) \
+    __ApplePrivate_gss_wrap_iov(t, u, v, w, x, y, z)
```

Comment:
I think it would be more clear to indent the continuation lines, unless that breaks something.

Reply:
Also: if this is actually just C, there shouldn't be a need to specify the arguments at all - one can just do e.g. `#define gss_wrap_iov __ApplePrivate_gss_wrap_iov`

Reply:
Nice, just tested it and it works just fine. This is my C noobness shining through.